content/en/docs/Concepts/DTO.md

```diff
@@ -1,11 +1,122 @@
 description: >
-  Lead in description for DTOs
+  Learn about data transfer objects (aka ViewModels in Asp.net MVC).
 ---
 
-## Overview
+## What are they?
+
+A data transfer object is a class that’s used for public display of another object in the app. DTO makes a class more presentable by hiding sensitive data.
+
+## Benefits of using them over raw entities?
+
+A layer of DTOs isolates the domain model from the presentation, resulting in both loose coupling and optimized data transfer. In loose coupling, you can change one service without changing the other by reducing interdependencies. If DTOs are used, a change in the requirements that forces a move to a different amount of data doesn't have any impact on the service layer or even the domain.
+
+## How does Asp.Net core prevent overposting attacks by using field binding?
+
+### - What is overposting?
+
+Overposting is a type of cyber attack that targets websites. More specifically, overposting targets models that expect an input from a user, but have other variables that are intentionally hidden from the user. For example, if a model has a “IsAdmin” variable that limits what a user can and cannot do, a hacker who knows this information can manually add an extra field to an input that would allow them to change the “IsAdmin” variable.
+
+### - How is this prevented?
+
+Take a look at an example set of code \(provided by [Scott Hanselmen](https://www.hanselman.com/blog/aspnet-overpostingmass-assignment-model-binding-security)\):
+
+```
+public class Person
+{
+    public int ID { get; set; }
+    public string First { get; set; }
+    public string Last { get; set; }
+    public bool IsAdmin { get; set; }
+}
+```
+
+```
+[HttpPost]
+[ValidateAntiForgeryToken]
+public async Task<IActionResult> Create(Person person)
+{
+```
+
+```
+ if (ModelState.IsValid)
+    {
+        _context.Add(person);
+        await _context.SaveChangesAsync();
+        return RedirectToAction("Index");
+    }
+    return View(person);
+}
+```
+
+Here we see an entity and a corresponding action that accepts the entity as a parameter. Let's say that the controller only wants the “First” and “Last” variables to be gathered from an input. A hacker can very easily add another field to an input that allows them to edit either the “ID” or the “IsAdmin” field. Now, look at the same code again but this time it is using binding.
+
+```
+[HttpPost]
+[ValidateAntiForgeryToken]
+public async Task<IActionResult> Create([Bind("First,Last")] Person person)
+{
+    if (ModelState.IsValid)
+    {
+        _context.Add(person);
+        await _context.SaveChangesAsync();
+        return RedirectToAction("Index");
+    }
+    return View(person);
+}
+```
+
+Now the action has bound the desired variables to itself. This means that it will only accept inputs that edit either the “First” or “Last” variables and will not take any inputs from anything else. This makes the other variables such as “IsAdmin” completely inaccessible to hackers.
```

Comment:
Change "itself" to "person". Also, specify what "it" is when you say "it will only accept inputs".
The word "edit" doesn't really suit the context here, because binding and/or DTOs can be used for Creating, as seen in the code snippet above. In that case, you aren't editing anything, but creating something.